src/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py

```diff
@@ -333,7 +335,8 @@ def __pre_initialize(cls) -> list:
         cls._authManager = AuthManager(cls._getCSAuthorizarionSection(cls._fullComponentName))
 
         if not (urls := cls._pre_initialize()):
-            sLog.warn("no target method found", f"{cls.__name__}")
+            # cls.log is not yet available
```

Comment:
But `cls.log` is available, because it is used in the next line?

Reply:
right, outdated comment, I remove it